article.html

```diff
@@ -0,0 +1,683 @@
+                              <img src="./assets/facebook.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="facebook">
+                            </div>
+                          </div>
+
+                          <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_512298232.coreimg{.width}.svg/1669130210337/insta.svg" data-asset="/content/dam/LeaderImpact/site-./assets/insta.svg" data-title="instagram" id="image-ae3671446d" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                              <img src="./assets/insta.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="instagram">
+                            </div>
+                          </div>
+
+                          <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1887433809.coreimg{.width}.svg/1669130230195/yt.svg" data-asset="/content/dam/LeaderImpact/site-./assets/yt.svg" data-title="YouTube" id="image-46d1bf6845" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                              <img src="./assets/yt.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="YouTube">
+                            </div>
+                          </div>
+
+                          <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1801542987.coreimg{.width}.svg/1669130241057/linkedin.svg" data-asset="/content/dam/LeaderImpact/site-./assets/linkedin.svg" data-title="LinkedIn" id="image-bb85cd5a39" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                              <img src="./assets/linkedin.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="LinkedIn">
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid article aem-GridColumn aem-GridColumn--default--12 layout-grid">
+            <div id="article-container" class="cmp-container">
+                <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
```

Comment:
Please remove all instances of `<div class="aem-Grid aem-Grid--12 aem-Grid--default--12 "></div>` and fix the styling accordingly. That `<div>` will not always be present and shouldn't be counted on for styling. Most, if not all, instances of `:first-child` in the CSS will likely need to be fixed.

Reply:
should i remove that div from all over the site or just from the child wrapper

Reply:
The child wrapper.